pkg/controller/storagecluster/reconcile.go

```diff
@@ -409,3 +420,45 @@ func newCephDaemonResources(custom map[string]corev1.ResourceRequirements) map[s
 
 	return resources
 }
+
+func (r *ReconcileStorageCluster) validateStorageCluster(instance *ocsv1.StorageCluster) (bool, error) {
```

Comment:
What would you think about making this function just tell us whether or not the instance is the active one or not?

Reply:
I think that's a good idea, since the only validation we are currently doing is checking if StorageCluster is Active.